crates/wasi-http/wit/deps/http/types.wit

```diff
@@ -322,14 +330,14 @@ interface types {
 
     /// Set the timeout for the initial connect to the HTTP Server. An error
     /// return value indicates that this timeout is not supported.
-    set-connect-timeout: func(duration: option<duration>) -> result;
+    set-connect-timeout: func(ms: option<duration>) -> result;
```

Comment:
lets not pull these `ms` names from upstream and instead fix those upstream?

Reply:
Whoops, thanks for catching this!